modules/tinymce/src/plugins/table/main/ts/actions/ResizeHandler.ts

```diff
@@ -93,48 +92,33 @@ export const getResizeHandler = function (editor: Editor): ResizeHandler {
   editor.on('ObjectResizeStart', function (e) {
     const targetElm = e.target;
     if (isTable(targetElm)) {
+      const table = Element.fromDom(targetElm);
 
-      const tableHasPercentage = Util.getRawWidth(editor, targetElm).exists(Util.isPercentage);
-
+      const tableHasPercentage = Sizes.isPercentSizing(table);
       if (tableHasPercentage && isPixelsForced(editor)) {
-        enforcePixels(targetElm);
+        enforcePixels(editor,table);
```

Comment:
Space missing? I'm surprised the linter didn't pick up on this...

Reply:
Thanks! I'll need to find out why eslint didn't flag this after 5.4 is out.

Reply:
Fixed